package.json

```diff
@@ -28,15 +29,15 @@
   "bundlesize": [
     {
       "path": "full/preact.js",
-      "maxSize": "750b"
+      "maxSize": "727b"
```

Comment:
the other `assign` function cost an additional 17 bytes, so we should keep the existing one - good call!